### PR TITLE
Log video play events

### PR DESCRIPTION
**Pull request by dadlerj**

### CODE REVIEW

File: src/pages/own.tsx [hunk 1]
```diff
@@ -107,6 +108,7 @@ const OwnPage: FunctionComponent = () => (
                     playsInline={true}
                     controls={false}
                     data-cookieconsent="ignore"
+                    onPlay={() => getEventLogger().log('StaticVideoPlayed', {title: 'Own landing page video 3'}, {title: 'Own landing page video 3'})}
```
Comment (nathan-downs) [inline src/pages/own.tsx:111]: Is there any way to extract the `own-landing-page-3` part out rather than having the video number hard coded?

Comment (dadlerj) [reply src/pages/own.tsx:111]: Honestly what I really want is an actual human-understandable title for the video. I'll just go check out the page and see what the video is!

**Review by nathan-downs — DISMISSED** [review]
Looks good to me

File: src/pages/own.tsx (hunk 1 above)
Comment (dadlerj) [reply src/pages/own.tsx]: Updated, thanks

**Pull request merged**